### DISCUSSION TITLE
Can i access pocketbase when i install an ssl certificate

### DISCUSSION THREAD

**Discussion by thal-sh** [pocketbase/pocketbase#4715] [Q&A] [answered]
my app used to work fine when i was hosting my pocketbase on digitalocean and it was http, now i switched to https and requests arent working anymore, im using nextjs and it is giving me stuff like this ClientResponseError 0: Something went wrong while processing your request. Is it a cors issue? not sure cant see any errors related to that

**Comment by thal-sh** [comment, marked as answer]
i fixed it, it was apparently something on my side, thanks

**Comment by vamshinenu** [comment]
hi there, how did you fix, it, like it would be helpful, to know the actual problems solution, thanks man

**Comment by thal-sh** [comment]
i usually do say how did I fix it but really it just was me forgetting not to update pocketbase instance to the new link haha pretty much it so it wasn't worth mentioning
